### DISCUSSION TITLE
Focus CSS/Style

### DISCUSSION THREAD

**Discussion by nvima** [BuilderIO/jsx-lite#971] [Q&A] [answered]
Is it possible to add something like this to an input with style property?
`textarea:focus, input:focus{
    outline: none;
}
`

**Comment by nvima** [comment, marked as answer]
Found it in an pull request.
https://github.com/BuilderIO/mitosis/pull/639

**Reply by samijaber** [reply]
@nvima Good find. We don't have thorough tests for this feature, so I am not 100% sure it works in every output.

**Reply by nvima** [reply]
Vue3 seems to work, if I notice something I will report it. Do you know how I can pass all props to a component in short form.
Something like that?
`
<Chat props={...props} />
`

**Reply by samijaber** [reply]
Yes, you can use standard JSX syntax for that: [see example](https://mitosis.builder.io/?outputTab=G4VwpkA%3D&code=JYWwDg9gTgLgBAYQBYEN4DMoRHA5AOgHpk18AbYGAU1wG4AoeqgD0ljgBMr0UBXMjLwB2AYxjAIQuAFkAngmyQhVITAAUYLGADOASjgBvenDhQqMXlClrjJuwB4S8GEmBCA5gF4DBuPn%2BaEDoANHAA1gBccACMAEwAzHAAvklwhAB8troMSUA%3D%3D%3D)
